util/ci/ubuntu-14.04-testscript

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash -e
+
+# -e above so that if any command has an exit code != 0, this script will
+#  exit immediately.
+
+mkdir -p /root/.local/share
+
+mkdir build;
+cd build;
+export TZ="America/Los_Angeles";
+
+ls -al /;
+ls -al /gtest;
```

Comment:
Considering you didn't add these lines to the arch testscript I assume the two lines were only necessary to debug your docker setup ?

Reply:
It's related to the way we get google test differently in ubuntu than arch linux. I'll remove this, too.